code_generator/End2End/schema/nested_typenames.schema

```diff
@@ -1,26 +1,24 @@
 package improbable.test_schema;
 
-// TODO: UTY-360 Uncomment this file when Nested Type definitions are supported.
-//
-// type NestedTypeName {
-//     type Other {
-//         type NestedTypeSameName {
-//             type Other0 {
-//                 int32 foo = 1;
-//             }
-//             enum NestedEnum {
-//                 enum_value = 1;
-//             }
-//             Other0 other_zero = 1;
-//             NestedEnum enum_field = 2;
-//         }
-//         NestedTypeSameName same_name = 1;
-//     }
-//     Other other = 1;
-// }
+type NestedTypeName {
+    type Other {
+        type NestedTypeSameName {
```

Comment:
I'm very confused by this file. What is this the same name as?

Reply:
Could we name the components based on what tier they are in, or something that makes it clearer what is meant to be going on here.

Reply:
No idea, I think this was copied from the WorkerSDK?

Reply:
Yeah why not